src/sbt-test/cucumber/scenario_outlines/src/test/scala/com/waioeka/sbt/MultiplicationSteps.scala

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2015, Michael Lewis
+ * All rights reserved.

```

Comment:
This seems problematic. If we're doing Lightbend CLA the copy right needs to transfer to Lightbend.